backend/app/main.py

```
@@ -1500,6 +2216,431 @@ def _process_and_store(file_path: Path, report_week: str, artifact_id: str, suff
+                            "warnings": payload.get("warnings"),
+                        }
+                    },
+                )
+                results.append({"filename": url, "status": "success", "facts_count": 0, "evidence_count": 1})
+            except Exception as exc:
+                results.append({"filename": url, "status": "error", "error": str(exc)})
+
+    return {"results": results}
+
+@app.get("/facts")
+async def get_facts(report_week: str = None):
```

Ruff (0.15.6)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)